packages/components/src/utils/unit-values.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Parses a number and unit from a value.
+ *
+ * @param  toParse Value to parse
+ *
+ * @return  The extracted number and unit.
+ */
+export function parseUnitValue(
```

Comment:
Nit (feel free to ignore) — since this function returns `[value, unit]`, should we call it `parseValueUnit` ?